todo_app/templates/index.html

```diff
@@ -23,6 +23,11 @@ <h2>Items</h2>
           <li>
             <a href="{{url_for('get', id=item.id)}}">{{ item.title }}
```

Comment:
This a tag seems to never be closed, which is leading to the HTML making some guesses about what to do - and it's conveniently guessing rightly but will probably confuse in future! The easiest fix is probably to update this to show the name & close it there, and get rid of the repeated name tag below
```suggestion
            <a href="{{url_for('get', id=item.id)}}">{{ item.name }} </a>
```